pandas-stubs/core/series.pyi

```diff
@@ -4466,7 +4461,7 @@ class Series(IndexOpsMixin[S1], NDFrame):
         level: None = ...,
         numeric_only: _bool = False,
         **kwargs: Any,
-    ) -> float: ...
+    ) -> Any: ...
```

Comment:
I'd rather leave this as `float`.  While it's true that the untyped `Series` coming from a column in a `DataFrame` could be a series of `Timestamp`, most people will want the type of the result of this to be `float`.